week1/practice/ComposeArticle/app/src/main/java/com/example/composearticle/MainActivity.kt

```diff
@@ -39,7 +54,37 @@ private fun ArticleCard(
     imagePainter: Painter,
     modifier: Modifier = Modifier,
 ) {
-    Column() { }
+    Column(
+        modifier=modifier.fillMaxSize(),
+        verticalArrangement = Arrangement.Top,
+    ) {
+        Image(
+            painter = imagePainter,
+            contentDescription = null,
+        )
+        Text(
+            text = title,
+            fontSize = 24.sp,
+            modifier = Modifier.padding(
+                all = 16.dp,
+            )
+        )
+        Text(
+            text = shortDescription,
+            textAlign = TextAlign.Justify,
+            modifier = Modifier.padding(
+                start = 16.dp,
+                end = 16.dp
+            )
```

Comment:
```suggestion
            modifier = Modifier.padding(horizontal = 16.dp)
```

마찬가지로 `horizontal`을 통해 `start`와 `end`를 동시에 지정할 수 있습니다!